stages/org.osbuild.bootc.install-to-filesystem

```diff
@@ -70,6 +77,9 @@ def main(options, inputs, paths):
                 tmpf.write(key.encode("utf8") + b"\n")
             tmpf.flush()
             pargs.extend(["--root-ssh-authorized-keys", tmpf.name])
+        # customize kernel-args
+        for karg in options.get("kernel-args", []):
+            pargs.extend(["--karg", karg])
```

Comment:
Can `--karg` be specified multiple times in `bootc`?  The docs aren't very clear.

Reply:
I'm about 80% confident it can but it's far from obvious from the docs or code so I opened https://github.com/containers/bootc/pull/392

Reply:
Looks like it works!  Thanks.